db/migrate/20161126190339_add_draft_to_events.rb

```diff
@@ -1,5 +0,0 @@
-class AddDraftToEvents < ActiveRecord::Migration
-  def change
-    add_column :events, :draft, :boolean
-  end
-end
```

Comment:
You won't want to remove any migrations ever. It also seems you forgot to add your own migration that would add the published field.